ang/civicase/activity/panel/directives/activity-panel.directive.html

```diff
@@ -1,4 +1,4 @@
-<div class="civicase__activity-panel" ng-controller="caseActivityCardController">
+<div class="civicase__activity-panel civicase__activity__status-{{ activity.status | lowercase }}" ng-controller="caseActivityCardController">
```

Comment:
Lets not add this class here, instead add it to the `.civicase__activity-panel__core_container`.
https://github.com/compucorp/uk.co.compucorp.civicase/blob/6fbb3f60e7b1f090370681b908b2a7a89df60ea8/ang/civicase/activity/panel/directives/activity-panel.directive.html#L106

`<div class="civicase__activity-panel__core_container civicase__activity-panel__core_container--{{ activity.status | lowercase }}"></div>`